examples/workers/main.go

```diff
@@ -5,17 +5,14 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/cryptopay-dev/yaga/logger/log"
 	"github.com/cryptopay-dev/yaga/workers"
 	"go.uber.org/atomic"
 )
 
 func main() {
-	log.Init()
-
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	w := workers.New(ctx)
+	w := workers.New(nil)
```

Comment:
![wat](https://user-images.githubusercontent.com/888650/38415145-3ab1bf38-3999-11e8-8a42-cfa2d2be7dda.jpg)

Reply:
we don't want test redis exclusive workers?

Reply:
It's not test. And example without exclusive workers ))

Reply:
Tests will be added later